tests/integration/models/os.spec.ts

```diff
@@ -336,6 +400,17 @@ describe('OS model', function () {
 				expect(draftVersions).to.have.length.greaterThan(0);
 			});
 
+			it('should include draft OS versions when the respective flag is used [string device type argument]', async () => {
```

Comment:
Isn't this testing *not* including draft OS versions? I don't see the `includeDraft` flag being used

Reply:
It seems that I totally forgot to change the title of the test case...